### PR TITLE
Adds PR Template

### PR DESCRIPTION
**Pull request by marcabisamra**
Title says it all

### CODE REVIEW

File: PULL_REQUEST_TEMPLATE.md [hunk 1]
```diff
@@ -0,0 +1,11 @@
+## Summary of Changes
+- A concise list of changes
+
+## Links
+- [Link to story]()
+- [Link to sandbox]()
+
+## How to test 
+- Concise instructions if unclear (optional)
```
Comment (mattfelsen) [inline PULL_REQUEST_TEMPLATE.md]: Small request to remove "optional". We should know how to test, and if it's unclear then instructions should be provided. :)

Comment (marcabisamra) [reply PULL_REQUEST_TEMPLATE.md]: For sure

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -21,16 +21,16 @@
...
 | Types   | Branch Name                           |
 | ------- | ------------------------------------- |
-| bug     | `bug/[branch-name]-#[pivotal-id]`     |
-| chore   | `chore/[branch-name]-#[pivotal-id]`   |
-| feature | `feature/[branch-name]-#[pivotal-id]` |
-| hotfix  | `hotfix/[branch-name]-#[pivotal-id]`  |
+| bug     | `bug/[branch-name]-[pivotal-id]`     |
+| chore   | `chore/[branch-name]-[pivotal-id]`   |
+| feature | `feature/[branch-name]-[pivotal-id]` |
+| hotfix  | `hotfix/[branch-name]-[pivotal-id]`  |
```
Comment (marcabisamra) [inline CONTRIBUTING.md:33]: We agreed on no `#` in the branch name, squeezed it in here

**Comment by MechanicalHuman** [comment]
What are the rules for merging branches?

**Pull request merged**